src/Verifier.php

```diff
@@ -30,6 +30,8 @@ final class Verifier implements Domain\Verifier
      */
     private $signer;
 
+    private $clockSkewSeconds = 5 * 60;
```

Comment:
Could you please rename the variable to `$leewayInSeconds` and set it to `300`? The line could then read as

```php
private $leewayInSeconds = 300; // 5 Minutes
```